maro/simulator/scenarios/cim/topologies/global_trade.22p_l0.8/config.yml

```diff
@@ -1,234 +1,230 @@
-seed: 4096
```

Comment:
@chaosddp @Jinyu-W  do we need this?

Reply:
ignore it, looks like the generated config file doesn't keep the order. Please double confirm it.